RHDEVDOCS-3866 Merging the accessing perspectives sections (#52924)
openshift/openshift-docs: modules/odc-accessing-perspectives.adoc

// Module included in the following assemblies:
//
// web_console/web-console-overview.adoc

:_content-type: PROCEDURE
[id="accessing-perspectives_{context}"]
= Accessing the Perspectives

You can access the *Administrator* and *Developer* perspective from the web console as follows:

.Prerequisites

> To access a perspective, ensure that you have logged in to the web console. Your default perspective is determined by your role.

Reviewer: not sure if we need to get into details of what these role is 

@jerolimov WDYT?

Reviewer 2: Yeah this might be interesting to some users, we've got this question also in the past from associates who uses the web console UI less often. Wdyt:

```suggestion
To access a perspective, ensure that you have logged in to the web console. Your default perspective is automatically determined by the permission of the users. The *Administrator* perspective is selected for users with access to all projects, while the *Developer* perspective is selected for users with limited access to just their own projects.
```

Reviewer: Thanks @jerolimov , this looks better IMO

Author: Thank you @jerolimov. Additionally I'm removing "just" from from the suggested changes.

.Additional Resources
See link:https://docs.openshift.com/container-platform/4.11/web_console/adding-user-preferences.html[Adding User Preferences] for more information on changing perspectives.

.Procedure

. Use the perspective switcher to switch to the *Administrator* or *Developer* perspective.

> . Select an existing project from the list or use the *Project* drop-down list to create a new project.

Reviewer 2: I don't understand what "from the list" means here. Esp. new users without access to any project will not have such a list. I want to suggest to focus here on the project dropdown:

```suggestion
. Select an existing project from the *Project* drop-down list. You can also create a new project from this dropdown.
```
